proto/proto-backplane-grpc/Dockerfile

```diff
@@ -84,4 +85,71 @@ RUN set -eux; \
     /includes/deephaven/proto/partitionedtable.proto \
     /includes/deephaven/proto/config.proto \
     /includes/deephaven/proto/hierarchicaltable.proto \
-    /includes/deephaven/proto/storage.proto;
+    /includes/deephaven/proto/storage.proto; \
+  # proto-doc-gen does not support writing multiple proto files to multiple docs, must break up command
```

Comment:
Do all of these files have proper crosslinking?  If not, we should generate a single md file with everything in it.

Also, should we be generating HTML docs as well since it is easy to do right now?  If so, should there be subdirs for md vs HTML output?